libpius/path.py

```diff
@@ -0,0 +1,124 @@
+'''A set of path functions and variables for the PIUS suite.'''
+from __future__ import print_function
+
+import os
+import sys
+import stat
+from os.path import abspath
+import fnmatch
+
+LINUX_TEMPDIR = '/tmp/'
+BIN_PATHS = '/usr/bin;/usr/sbin/;/bin;/sbin;/usr/local/bin;/usr/local/sbin'
```

Comment:
Some versions of linux uses /usr/local so it needs to stay, it will be difficult to hard code for every little difference between different variants. Also the constant holds as the name implies all the bin paths.

Reply:
No distribution installs software into /usr/local - that's now how FSB works. /usr/local is where user-compiled software goes, which is not where we'd expect security-sensitive software like GPG to be on a Linux machine. We should _not_ look in /usr/local by default on windows unless to user asks us to. Remember the user can do so with a config file by adding `gpg-path=/usr/local/bin/gpg` to their `.piusrc`. That's **why** we have a config file for anyone with a non-standard configuration.

Reply:
I can make the change it will however need to have the 2 joined for macOS since it could be built on the machine or it could be in normal bin if it is an older version of OSX.